### DISCUSSION TITLE
Default Values

### DISCUSSION THREAD

**Discussion by BothimTV** [SudhanPlayz/Discord-MusicBot#132] [Ideas]
Hi, can u add an Feature, that allows you to set defaults. Like Def. Volume = 10 or Def. Loop = true. That would be nice <3

**Comment by ItzSudhan** [comment]
you can change it while creating player of the guild

**Reply by fred8044** [reply]
Hey, how to set volume default level : 3 ? what's guild ? thanks

**Reply by brianferri** [reply]
The default volume can be set in the config file, using values from 1 to 100, in the root directory of the bot. In your case you'd probably want to set the volume to 30.
A "guild" is a _discord server_.

**Reply by fred8044** [reply]
thanks :)
